performance-test/locustfile.py

```
@@ -0,0 +1,38 @@
+import csv
+import random
+from locust import HttpUser, between
+from tasks.notification_tasks import NotificationTasks
+#from tasks.파일명 import 임포트명 형식으로 작성
+
+# 테스트 유저 리스트
+user_id_pool = []
+
+# 테스트 시작전 유저 목록 불러오기
+try:
+    with open('data/users.csv', 'r') as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)